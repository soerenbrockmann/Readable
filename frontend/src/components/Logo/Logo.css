.logo {
  display: flex;
  justify-content: row;
  font-size: 36pt;
}

.firstletter {
  color: firebrick;
}

.logolink,
a {
  text-decoration: none;
  color: #000;
}
