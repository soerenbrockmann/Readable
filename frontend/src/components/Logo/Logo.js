import React from "react";
import { Link } from "react-router-dom";
import "./Logo.css";

const Logo = () => {
  return (
    <Link to={"/"} className="logolink">
      <div className="logo">
        <div className="firstletter">R</div>
        <div>eadable</div>
      </div>
    </Link>
  );
};

export default Logo;
