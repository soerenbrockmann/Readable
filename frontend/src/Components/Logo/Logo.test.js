import React from "react";
import ReactDOM from "react-dom";
import { shallow } from "enzyme";
import Logo from "./Logo";

describe("<Logo />", () => {
  const logo = shallow(<Logo />);

  it("renders properly", () => {
    expect(logo).toMatchSnapshot();
  });
});
