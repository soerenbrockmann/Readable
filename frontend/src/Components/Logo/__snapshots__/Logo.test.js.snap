// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Logo /> renders properly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Logo />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "class",
    "props": Object {
      "children": <div
        className="logo"
>
        <div
                className="firstletter"
        >
                R
        </div>
        <div>
                eadable
        </div>
</div>,
      "className": "logolink",
      "replace": false,
      "to": "/",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <div
            className="firstletter"
>
            R
</div>,
          <div>
            eadable
</div>,
        ],
        "className": "logo",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "R",
            "className": "firstletter",
          },
          "ref": null,
          "rendered": "R",
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "eadable",
          },
          "ref": null,
          "rendered": "eadable",
          "type": "div",
        },
      ],
      "type": "div",
    },
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "children": <div
          className="logo"
>
          <div
                    className="firstletter"
          >
                    R
          </div>
          <div>
                    eadable
          </div>
</div>,
        "className": "logolink",
        "replace": false,
        "to": "/",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <div
              className="firstletter"
>
              R
</div>,
            <div>
              eadable
</div>,
          ],
          "className": "logo",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "R",
              "className": "firstletter",
            },
            "ref": null,
            "rendered": "R",
            "type": "div",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "eadable",
            },
            "ref": null,
            "rendered": "eadable",
            "type": "div",
          },
        ],
        "type": "div",
      },
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
